data/wanderer/wanderers start.txt

```diff
@@ -224,10 +224,12 @@ mission "Wanderers: Diplomacy"
 
 
 mission "Wanderers: Translation Machine"
-	description "Sayari, the Hai ambassador, suggested that you visit <destination>, where Hai technologists are working on translation machines."
+	description "Find language training data at <destination> so the Wanderers can create an automated language translator."
 	landing
 	source "Vara K'chrai"
-	destination "Greenwater"
+	destination "Alexandria"
+	substitutions
+		"<Library search>" `Traveling through the library in search for data suitable for training the Wanderer translation algorithm raises an unexpected problem: you're surrounded by it. You guessed that even fifty Bulk Freighters wouldn't be enough to carry all the tomes and data cores from the Alexandria library. Asking around, you're referred to a research librarian who develops automated translation algorithms for dead languages.`
```

Comment:
One issue here, we've already established that you can get basically the sum of human historical data on a storage you can carry yourself.
Right now, you can store well more data cold than you would ever need for training translation in media you could fit into a backpack.

I think this should be a matter where you should need to acquire a set of data that is comprehensive but structured, not just a carte blanche of content that might be centuries out of date, linguistically.

I do not know if the player might have a way of being told to look for this specifically, or if it might be something which requires you to make multiple trips because the first data batch only works well enough for you to realise there's an issue and have them explain better criteria for data to bring back.

Reply:
Whatever is in there will be in the language of modern day since it is intended to be understandable by people who know only the common language. That large of a quantity of data is easily enough to train a machine learning algorithm to translate.